app/metrics.yaml

```diff
@@ -509,6 +509,220 @@ metrics:
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  open_links_in_a_private_tab:
+    type: string_list
+    description: >
+      Whether or not the user has enabled open links in a private tab.
+      default: false
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  sync:
+    type: string_list
+    description: >
+      Whether or not the user is signed into FxA
+      default: false
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  sync_items:
+    type: string_list
+    description: >
+      The list of items the user has chosen to sync with FxA.
+      default: "" if the user is signed out. Otherwise defaults to:
+      [bookmarks, history, passwords, tabs]
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  show_voice_search:
+    type: string_list
+    description: >
+      Whether or not the user has enabled the voice search button.
+      default: true
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  search_suggestions_private:
+    type: string_list
+    description: >
+      Whether or not the user has enabled showing search suggestions
+      in private mode.
+      default: false (we prompt the user, asking them to make a selection)
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  toolbar_position:
+    type: string_list
+    description: >
+      The position of the toolbar
+      default: bottom (defaults to top if the user has accessibility services)
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  accessibility_services:
+    type: string_list
+    description: >
+      Whether or not the user has touch exploration or switch services enabled
```

Comment:
Nit: would clarify that this is on the Android system, bc by default might assume Fenix.